packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -218,8 +220,9 @@ const SettingsSidebarContainer = ({
 
   return (
     <nav
+      style={{ maxHeight: `calc(100vh - ${bannersHeight}px)`, top: `${bannersHeight}px` }}
       className={classNames(
-        "no-scrollbar bg-muted fixed bottom-0 left-0 top-0 z-20 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:flex",
```

Comment:
don't remove no-scrollbar ( not sure about using overflow-y-auto).
It causes this bug in brave browser

<img width="223" alt="Screenshot 2023-07-31 at 12 07 13 PM" src="https://github.com/calcom/cal.com/assets/53316345/6ea326b4-9cff-497b-a7ca-43ffa0e4d5f9">
llbar

Reply:
Hi @Udit-takkar 
We are having the same classes for main page sidebar, so brave browser issue should there as well.
Is it still ok ?

Reply:
Did the changes now. Thanks @Udit-takkar